src/modules/update/Kconfig

```diff
@@ -0,0 +1,12 @@
+menuconfig USE_AUTO_UPDATE
+    bool "Auto Update Options"
```

Comment:
In the beginning I thought this as 'an auto update' because requires no final user intervention - but I dropped the idea and the 'auto' thing survived only here =D